data/yaml/venues/wsc.yaml

```diff
@@ -0,0 +1,2 @@
+acronym: wsc
```

Comment:
The acronym is how it's displayed, should probably be `WSC`.

Also is there a website?